### PR TITLE
Switched MQTT go library to support MQTT 5.

### PR DESCRIPTION
**Pull request by KairaMiu**

### CODE REVIEW

**Review by KlausPopp — CHANGES_REQUESTED** [review]
Please add in Makefile tests to .PHONY: 
.PHONY: clean example test

git commit hook missing

File: alm-mqtt-module/main.go [hunk 1]
```diff
@@ -103,19 +105,31 @@ func main() {
 	natsClient = <-natsClientChan
```
[the remark this answers is not in the document]
Comment (KairaMiu) [reply alm-mqtt-module/main.go:105]: Done

File: alm-mqtt-module/main.go [hunk 2]
```diff
@@ -103,19 +105,31 @@ func main() {
...
+		client := paho.NewClient(paho.ClientConfig{
+			Conn:   conn,
+			Router: paho.NewSingleHandlerRouter(mqttHandler),
+		})
+
 		for i := 0; i < connectTimeoutSeconds; i++ {
-			if token := client.Connect(); token.Wait() && token.Error() != nil {
-				log.Printf("Connect failed: %s\n", token.Error())
-				log.Printf("Reconnecting to '%s'\n", mqttServer)
+			// Connect Client to MQTT Broker
```
[the remark this answers is not in the document]
Comment (KairaMiu) [reply alm-mqtt-module/main.go:124]: Done

File: alm-mqtt-module/main.go [hunk 3]
```diff
@@ -138,21 +152,34 @@ func main() {
...
 		case removeMqttTopic := <-newConfigUnregisterChan:
 			fmt.Printf("Unsubscribing '%s'\n", removeMqttTopic)
-			if token := (*mqttClient).Unsubscribe(removeMqttTopic); token.Wait() && token.Error() != nil {
-				log.Fatal(token.Error())
+			if _, err := (*mqttClient).Unsubscribe(context.Background(), &paho.Unsubscribe{
+				Topics: []string{removeMqttTopic},
+			}); err != nil {
+				log.Fatal(err)
 			}
+			// mqttClient.Router.UnregisterHandler(removeMqttTopic)
```
[the remark this answers is not in the document]
Comment (KairaMiu) [reply alm-mqtt-module/main.go:172]: Removed

**Review by KlausPopp — APPROVED** [review]

**Pull request merged**